pennathletics/athletes.py

```diff
@@ -17,6 +17,28 @@ def get_roster(sport, year):
             parse.parse_hometown(person[6]))
         )
     return roster
-        
     
-# print(get_roster('M_Basketball',2015))
\ No newline at end of file
+def get_player(sport, year, name = "", no = -1):
```

Comment:
PEP8, no spaces around `=` in optional function arguments. Use pyflakes or pylint or whatever code editor plugin to make sure your code is clean before code review.